<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="2" ignored="0" total="5" passed="3">
  <reporter-output>
    <line>
      <![CDATA[RummyBaazi Site Opens]]>
    </line>
  </reporter-output>
  <suite name="Suite" duration-ms="116757" started-at="2021-04-20T19:11:45 IST" finished-at="2021-04-20T19:13:42 IST">
    <groups>
    </groups>
    <test name="Login" duration-ms="16518" started-at="2021-04-20T19:11:45 IST" finished-at="2021-04-20T19:12:02 IST">
      <class name="testCases.login_TestCases">
        <test-method status="PASS" signature="Login()[pri:0, instance:testCases.login_TestCases@431cd9b2]" name="Login" duration-ms="12347" started-at="2021-04-20T19:11:45 IST" finished-at="2021-04-20T19:11:58 IST">
          <reporter-output>
            <line>
              <![CDATA[RummyBaazi Site Opens]]>
            </line>
          </reporter-output>
        </test-method> <!-- Login -->
        <test-method status="PASS" signature="after()[pri:0, instance:testCases.login_TestCases@431cd9b2]" name="after" is-config="true" duration-ms="4111" started-at="2021-04-20T19:11:58 IST" finished-at="2021-04-20T19:12:02 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- after -->
      </class> <!-- testCases.login_TestCases -->
    </test> <!-- Login -->
    <test name="profile" duration-ms="17863" started-at="2021-04-20T19:12:02 IST" finished-at="2021-04-20T19:12:20 IST">
      <class name="testCases.profile_TestCases">
        <test-method status="PASS" signature="Login()[pri:0, instance:testCases.profile_TestCases@23f5b5dc]" name="Login" is-config="true" duration-ms="11734" started-at="2021-04-20T19:12:02 IST" finished-at="2021-04-20T19:12:14 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Login -->
        <test-method status="PASS" signature="Profile()[pri:0, instance:testCases.profile_TestCases@23f5b5dc]" name="Profile" duration-ms="6125" started-at="2021-04-20T19:12:14 IST" finished-at="2021-04-20T19:12:20 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Profile -->
        <test-method status="PASS" signature="after()[pri:0, instance:testCases.profile_TestCases@23f5b5dc]" name="after" is-config="true" duration-ms="0" started-at="2021-04-20T19:12:20 IST" finished-at="2021-04-20T19:12:20 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- after -->
      </class> <!-- testCases.profile_TestCases -->
    </test> <!-- profile -->
    <test name="deposit" duration-ms="17279" started-at="2021-04-20T19:12:20 IST" finished-at="2021-04-20T19:12:37 IST">
      <class name="testCases.deposite_TestCases">
        <test-method status="PASS" signature="Login()[pri:0, instance:testCases.deposite_TestCases@7857fe2]" name="Login" is-config="true" duration-ms="12939" started-at="2021-04-20T19:12:20 IST" finished-at="2021-04-20T19:12:33 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Login -->
        <test-method status="FAIL" signature="Deposit()[pri:0, instance:testCases.deposite_TestCases@7857fe2]" name="Deposit" duration-ms="3771" started-at="2021-04-20T19:12:33 IST" finished-at="2021-04-20T19:12:37 IST">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Enter the Amount less than 50000.00] but found [Enter the Amount less than 1000.00]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Enter the Amount less than 50000.00] but found [Enter the Amount less than 1000.00]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at pageFactory.deposit_POF.maxamountValidation(deposit_POF.java:55)
at testCases.deposite_TestCases.Deposit(deposite_TestCases.java:44)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1259)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Deposit -->
        <test-method status="PASS" signature="after()[pri:0, instance:testCases.deposite_TestCases@7857fe2]" name="after" is-config="true" duration-ms="220" started-at="2021-04-20T19:12:37 IST" finished-at="2021-04-20T19:12:37 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- after -->
      </class> <!-- testCases.deposite_TestCases -->
    </test> <!-- deposit -->
    <test name="withdrawal" duration-ms="39284" started-at="2021-04-20T19:12:37 IST" finished-at="2021-04-20T19:13:17 IST">
      <class name="testCases.withdrawal_TestCases">
        <test-method status="PASS" signature="Login()[pri:0, instance:testCases.withdrawal_TestCases@5340477f]" name="Login" is-config="true" duration-ms="13355" started-at="2021-04-20T19:12:37 IST" finished-at="2021-04-20T19:12:51 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Login -->
        <test-method status="FAIL" signature="Withdrawal()[pri:0, instance:testCases.withdrawal_TestCases@5340477f]" name="Withdrawal" duration-ms="21664" started-at="2021-04-20T19:12:51 IST" finished-at="2021-04-20T19:13:12 IST">
          <exception class="org.openqa.selenium.NoSuchElementException">
            <message>
              <![CDATA[no such element: Unable to locate element: {"method":"xpath","selector":"//button[contains(text(),'Continue')]"}
  (Session info: chrome=90.0.4430.72)
For documentation on this error, please visit: https://www.seleniumhq.org/exceptions/no_such_element.html
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:17:03'
System info: host: 'LAPTOP-FET714UI', ip: '192.168.0.104', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_261'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 90.0.4430.72, chrome: {chromedriverVersion: 89.0.4389.23 (61b08ee2c5002..., userDataDir: C:\Users\rshiv\AppData\Loca...}, goog:chromeOptions: {debuggerAddress: localhost:50885}, javascriptEnabled: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: WINDOWS, platformName: WINDOWS, proxy: Proxy(), setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:largeBlob: true, webauthn:virtualAuthenticators: true}
Session ID: 73ecbf6eab66ac213197e604092c85a4
*** Element info: {Using=xpath, value=//button[contains(text(),'Continue')]}]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.openqa.selenium.NoSuchElementException: no such element: Unable to locate element: {"method":"xpath","selector":"//button[contains(text(),'Continue')]"}
  (Session info: chrome=90.0.4430.72)
For documentation on this error, please visit: https://www.seleniumhq.org/exceptions/no_such_element.html
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:17:03'
System info: host: 'LAPTOP-FET714UI', ip: '192.168.0.104', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_261'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 90.0.4430.72, chrome: {chromedriverVersion: 89.0.4389.23 (61b08ee2c5002..., userDataDir: C:\Users\rshiv\AppData\Loca...}, goog:chromeOptions: {debuggerAddress: localhost:50885}, javascriptEnabled: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: WINDOWS, platformName: WINDOWS, proxy: Proxy(), setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:largeBlob: true, webauthn:virtualAuthenticators: true}
Session ID: 73ecbf6eab66ac213197e604092c85a4
*** Element info: {Using=xpath, value=//button[contains(text(),'Continue')]}
at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.createException(W3CHttpResponseCodec.java:187)
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:122)
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:49)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:158)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:83)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:552)
at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:323)
at org.openqa.selenium.remote.RemoteWebDriver.findElementByXPath(RemoteWebDriver.java:428)
at org.openqa.selenium.By$ByXPath.findElement(By.java:353)
at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:315)
at org.openqa.selenium.support.pagefactory.DefaultElementLocator.findElement(DefaultElementLocator.java:69)
at org.openqa.selenium.support.pagefactory.internal.LocatingElementHandler.invoke(LocatingElementHandler.java:38)
at com.sun.proxy.$Proxy27.click(Unknown Source)
at pageFactory.withdrawal_POF.emptyAmount(withdrawal_POF.java:31)
at testCases.withdrawal_TestCases.Withdrawal(withdrawal_TestCases.java:40)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1259)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.openqa.selenium.NoSuchElementException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Withdrawal -->
        <test-method status="PASS" signature="after()[pri:0, instance:testCases.withdrawal_TestCases@5340477f]" name="after" is-config="true" duration-ms="4069" started-at="2021-04-20T19:13:13 IST" finished-at="2021-04-20T19:13:17 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- after -->
      </class> <!-- testCases.withdrawal_TestCases -->
    </test> <!-- withdrawal -->
    <test name="changePassword" duration-ms="25269" started-at="2021-04-20T19:13:17 IST" finished-at="2021-04-20T19:13:42 IST">
      <class name="testCases.changePassword_TestCases">
        <test-method status="PASS" signature="Login()[pri:0, instance:testCases.changePassword_TestCases@126253fd]" name="Login" is-config="true" duration-ms="12210" started-at="2021-04-20T19:13:17 IST" finished-at="2021-04-20T19:13:29 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Login -->
        <test-method status="PASS" signature="ChangePassword()[pri:0, instance:testCases.changePassword_TestCases@126253fd]" name="ChangePassword" duration-ms="8887" started-at="2021-04-20T19:13:29 IST" finished-at="2021-04-20T19:13:38 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ChangePassword -->
        <test-method status="PASS" signature="after()[pri:0, instance:testCases.changePassword_TestCases@126253fd]" name="after" is-config="true" duration-ms="4169" started-at="2021-04-20T19:13:38 IST" finished-at="2021-04-20T19:13:42 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- after -->
      </class> <!-- testCases.changePassword_TestCases -->
    </test> <!-- changePassword -->
  </suite> <!-- Suite -->
</testng-results>
